data/events/2024-nairobi.yml

```
@@ -0,0 +1,104 @@
+name: "2024-nairobi" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2024" # The year of the event. Make sure it is in quotes.
+city: "NAIROBI" # The displayed city name of the event. Capitalize it.
```

As a stylistic choice, we tend to have cities use Title case, not allcaps
```suggestion
city: "Nairobi" # The displayed city name of the event. Capitalize it.
```

---

Fixed in https://github.com/devopsdays/devopsdays-web/pull/13542/commits/1c0018a8a3bc0d24f2c3afd5653dd638712c615f